MekHQ/src/mekhq/gui/dialog/nagDialogs/UnmaintainedUnitsNagDialog.java

```diff
@@ -50,10 +50,10 @@ static boolean checkHanger(Campaign campaign) {
     }
 
     /**
-     * Constructs a new UnmaintainedUnitsNagDialog.
+     * Creates a new instance of the UnmaintainedUnitsNagDialog class.
      *
-     * @param frame    the parent JFrame for the dialog
-     * @param campaign the Campaign object associated with the dialog
+     * @param frame the parent JFrame for the dialog
+     * @param campaign the Campaign associated with the dialog
```

Comment:
Use the JavaDoc code for linking to the Campaign class.

Reply:
I had no idea {@link} existed, thanks for the tip!